ReactAndroid/build.gradle

```diff
@@ -277,6 +277,14 @@ def reactNativeInspectorProxyPort() {
     return value != null ? value : reactNativeDevServerPort()
 }
 
+def reactNativeArchitectures() {
+    def isDebug = gradle.startParameter.taskRequests.any {
```

Comment:
There doesn't seem to be a simple way to check for debug builds in this code, but this works. The flag should never be set for release builds anyway but this helps in case someone sets it in their gradle.properties.

Reply:
This method is used with `ndk-build` so the debug `abiFilters` won't apply here.